<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCategoryDialog</class>
 <widget class="QDialog" name="AddCategoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="uButtonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>329</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="uVerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="uTitleHorizontalLayout">
      <item>
       <widget class="QLabel" name="uTitleLabel">
        <property name="text">
         <string> Title            </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uTitleForm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="uTagHorizontalLayout">
      <item>
       <widget class="QLabel" name="uTagLabel">
        <property name="text">
         <string> Tag             </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uTagForm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="uDescriptionHorizontalLayout">
      <item>
       <widget class="QLabel" name="uDescriptionLabel">
        <property name="text">
         <string> Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="uDescriptionForm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCategoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCategoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
